aptos-move/framework/aptos-framework/sources/managed_coin.move

```diff
@@ -97,6 +99,32 @@ module aptos_framework::managed_coin {
         coin::register<CoinType>(account);
     }
 
+    /// Destroys capabilities from the account, so that the user no longer has access to mint or burn.
+    public entry fun destroy_caps<CoinType>(account: &signer) acquires Capabilities {
+        let (burn_cap, freeze_cap, mint_cap) = remove_caps<CoinType>(account);
+        destroy_burn_cap(burn_cap);
+        destroy_freeze_cap(freeze_cap);
+        destroy_mint_cap(mint_cap);
+    }
+
+    /// Removes capabilities from the account to be stored or destroyed elsewhere
+    public fun remove_caps<CoinType>(
+        account: &signer
+    ): (BurnCapability<CoinType>, FreezeCapability<CoinType>, MintCapability<CoinType>) acquires Capabilities {
+        let account_addr = signer::address_of(account);
+        assert!(
+            exists<Capabilities<CoinType>>(account_addr),
```

Comment:
Would be nice to have a public function to view if `Capabilities ` exists in an address.